controllers/replace_failed_pods.go

```diff
@@ -48,12 +48,23 @@ func (c ReplaceFailedPods) Reconcile(r *FoundationDBClusterReconciler, context c
 
 // chooseNewRemovals flags failed processes for removal and returns an indicator
 // of whether any processes were thus flagged.
-func chooseNewRemovals(cluster *fdbtypes.FoundationDBCluster) bool {
+func chooseNewRemovals(r *FoundationDBClusterReconciler, cluster *fdbtypes.FoundationDBCluster) bool {
 	logger := log.WithValues("namespace", cluster.Namespace, "cluster", cluster.Name, "reconciler", "ReplaceFailedPods")
 	if !*cluster.Spec.AutomationOptions.Replacements.Enabled {
 		return false
 	}
 
+	// If we are not able to replace any failed instance because the fault tolerance dropped
+	// we don't want to block any subsequent steps.
+	hasDesiredFaultTolerance, err := r.hasDesiredFaultTolerance(cluster)
+	if err != nil {
+		return false
+	}
+
+	if !hasDesiredFaultTolerance {
+		return false
+	}
```

Comment:
Thinking more about this I think you're right John. With the assumption that the exclude command will block until the data is replicated + the check to only delete/remove Pods when we meet our fault tolerance (or a fully replicated) we shouldn't need this check here for the following reasons:

- The automatic replacements will be limited by the number of concurrent replacements (default `1`)
- We will only remove the process group and all associated resources (Pod, PVC, ..) when the cluster meets the fault tolerance req. (or is fully replicated)